### ISSUE TITLE
Issue with Open/Close Menu

### ISSUE THREAD

**Issue by Tubishi** [driftyco/ionic#6548]
Hi :), I have a problem with my Menu on Android, when i want to catch the event "menu.isOpen" nothing appear. I will explain better, but before my code.

```
let menu = this.app.getComponent('menu');
menu.getContentElement().addEventListener('click', () => {
		console.log(menu.isOpen);
	}
});
```
With iOS there is no problem, i have a good alternation between true and false, but on Android, i have always the same boolean, do you have any ideas ? Thanks for your answer.

Ionic2 Beta6

Cordova CLI: 6.1.1
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-beta.6
Ionic CLI Version: 2.0.0-beta.25
Ionic App Lib Version: 2.0.0-beta.15
OS: Distributor ID:	Ubuntu Description:	Ubuntu 15.04 
Node Version: v4.0.0

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! Would you mind making a plunker that i can use to reproduce this issue? Thanks!

**Comment by Tubishi** [comment]
I don't find the good way to show you http://plnkr.co/edit/Xj31MjTtTDphPYOsZYn9?p=preview, it's just a menu and when i Open/Close i put a console.log, with menu.isOpen, and with an Iphone i have false/true/false/true etc.... but Android just False/ False/ False/ etc....

**Comment by armand1m** [comment]
Hi there! I'm having trouble with these methods too. Seems like the `open` and `close` methods in the `MenuController` always resolve their promises to `false`. I looked on the code and saw that the `Menu` class calls and returns `this.setOpen(boolean)` when opening/closing the menu, so I got a promise resolved with the `this.isOpen` boolean value from the `Menu` class, but it always returns false. I saw that the `private _after(isOpen: boolean)` method in the `Menu` class its the one that actually updates its value, but only if this condition is evaluated to true `((this._isEnabled && isOpen) || !isOpen)`. I couldn't keep looking forward to found what may be causing that, but I just cant know through this promises in my code if the menu is really open or not.
